package.json

```diff
@@ -33,7 +33,6 @@
   "devDependencies": {
     "chai": "^3.5.0",
     "eslint": "^2.12.0",
-    "babel-eslint": "^7.2.1",
```

Comment:
I am afraid this change is off-topic. Could you raise another PR to remove babel-eslint dependencies?